unblob/file_utils.py

```diff
@@ -426,6 +427,9 @@ def has_root_permissions(self):
     def _fs_path(self, path: Path) -> _FSPath:
         return _FSPath(root=self.root, path=path)
 
+    def _ensure_parent_dir(self, path: Path):
```

Comment:
do we want to always do this, even if the extracted chunk is broken and actually miss that filesystem or just handle the "wrong order" case?

I am ok both, just want to make sure that it is a conscious decision

Reply:
These were the reasons:
- for properly working handlers (ordering is right) it is safe to do
- for handlers that are dealing with formats that do not necessarily have directory entries, this makes the code much simpler